expression/builtin_arithmetic.go

```diff
@@ -65,7 +65,7 @@ func handleDivisionByZeroError(ctx context.Context) error {
 		if (ctx.GetSessionVars().SQLMode & mysql.ModeErrorForDivisionByZero) != mysql.ModeErrorForDivisionByZero {
```

Comment:
define a function like `func(m SQLMode)HasErrorForDivisionByZeroMode() bool` in mysql/ const.go